### DISCUSSION TITLE
Can I get isPending for the v7_startTransition option?

### DISCUSSION THREAD

**Discussion by Jungho-Cheon** [remix-run/react-router#11987] [Q&A]
Hello.
I dynamically imported the code for page components through React.lazy, and I handled page switching by activating the v7_startTransition option to prevent flicker between page switching.
The problem is that the flicker phenomenon between page switches could be prevented, but there is no way to make additional configurations such as loading spinners. Is there a way to get react-router-dom internally managed isPending (such as the first useTransition return value..) when the v7_startTransition option is set?

**Comment by ha1fstack** [comment]
https://www.npmjs.com/package/react-router-topbar

If you need loading state for top loading bar implementation, I made package for the exact purpose.
